token-service/src/main/java/org/p2p/token/service/manager/TokenServiceEventListener.kt

```diff
@@ -0,0 +1,21 @@
+package org.p2p.token.service.manager
+
+import org.p2p.token.service.model.TokenServiceNetwork
+
+interface TokenServiceEventListener {
+    fun onUpdate(eventType: TokenServiceEventType, event: TokenServiceEvent)
+}
+
+enum class TokenServiceEventType {
+    SOLANA_CHAIN_EVENT,
```

Comment:
TokenServiceEventNetworkType

Reply:
I think it's ok, cause network name equals to type of event